### DISCUSSION TITLE
Student Pico is Unresponsive

### DISCUSSION THREAD

**Discussion by russlindell** [gizmo-platform/gizmo#27] [Q&A] [answered]
I'm a mentor who was setting up the motors on the Gizmo for a demonstration for the students, and it looks like I got something wrong. 

The student Pico got very hot and is no longer responsive. I unplugged everything and waited 15 minutes. When I connect to the mico USB slot,  the LED does not light up, and the board does not register on my laptop. When I connect to the system board, everything works as expected. So I'm assuming the issue is with this single Pico board. I visually inspected the board and I do not see any obvious issues but I'm assuming I killed the voltage regulator.

Question: Can I purchase a new Pico H and replace the damaged one? Is there anything in the rules stating otherwise?

**Comment by the-maldridge** [comment, marked as answer]
This sounds like a matter of rules for BEST Robotics.  All Rules questions should be directed to [BEST's Official Q&A](https://game.bestrobotics.org/qna/) as we are unable to answer them here.  That being said, yes if you killed a Pico H it will need to be replaced and I can't think of any reason that a Hub would not permit you to do that.

**Reply by Vaelatern** [reply]
I would further note there has been no discussion of the LEDs on the Gizmo in this question. This implies that the board power is not relevant here, despite driving motors.
